src/Selector.js

```diff
@@ -62,7 +74,11 @@ export default class Selector {
       else if (selected.isBillboard && selected.parent) {
         data = selected.parent.data;
       }
+      else if (selected.type === 'Mesh' && selected.parent.parent) {
+        data = selected.parent.parent.data;
```

Comment:
`data` will be undefined here. `selected.parent` will be a planet and `selected.parent.parent` the scene.